sort_function.py

```
@@ -0,0 +1,35 @@
+            return (1, int(num), name)
+      
+    numbers = [int(n) for n in NUMBERS.findall(stem)]
+    if numbers:
+        return (2 + min(numbers), min(numbers), name)
+       
+    return (10, 0, name)
+
+
+def choose_cover(files: list[str]) -> str:
+    if not files:
+        raise ValueError("Empty file list")
```

Ruff (0.11.9)

Avoid specifying long messages outside the exception class

(TRY003)